database/migrations/2016_09_06_124728_add_soft_deletes_to_posts_topics_tables.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddSoftDeletesToPostsTopicsTables extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('phpbb_posts', function (Blueprint $table) {
+            $table->softDeletes();
```

Comment:
that would be confusing. Something will be done for this... I think. Just add the index for now.